packages/hoppscotch-common/src/helpers/import-export/import/openapi.ts

```diff
@@ -74,11 +74,12 @@ const isOpenAPIV3Document = (
   )
 }
 
+// Make validation less strict for basic OpenAPI documents
 const hasRequiredOpenAPIFields = (doc: unknown): boolean => {
   return (
     objectHasProperty(doc, "info") &&
-    objectHasProperty(doc.info, "title") &&
-    objectHasProperty(doc.info, "version") &&
+    (objectHasProperty(doc.info, "title") ||
+      objectHasProperty(doc.info, "version")) &&
```

Comment:
Where is this function used ? It was used in `isABasicOpenAPIDoc`. but as i can see, `isABasicOpenAPIDoc`'s usage is removed in the PR.